.env.mainnet

```diff
@@ -71,7 +71,7 @@ REACT_APP_HASURA_URL=http://localhost:8080/v1/graphql
 REACT_APP_EOS_API_NETWORK_NAME=mainnet
 REACT_APP_EOS_API_NETWORK_LABEL=Mainnet
 REACT_APP_EOS_API_NETWORK_LOGO=https://antelope.tools/images/eos.png
-REACT_APP_EOS_API_HOST=eos.edenia.cloud
+REACT_APP_EOS_API_HOSTS=["eos.api.eosnation.io","eos.edenia.cloud","eos.greymass.com","eos.eosphere.io","api.eosrio.io"]
```

Comment:
i find the aloha EOS endpoint to be very good and dosen't rate limit as much. can we add it to the list: https://api.main.alohaeos.com